OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -110,7 +110,7 @@ bool IsAcceptableProcType(Actor proc)
 				info.DeliveryBuildings.Contains(proc.Info.Name);
 		}
 
-		Actor ClosestProc(Actor self, Actor ignore)
+		public Actor ClosestProc(Actor self, Actor ignore)
```

Comment:
That's something I was considering, yet given the weight that this function has and the usefulness that it may have, and how specific its specification is (get the closest Proc for a given actor) I considered it would be useful from a testing standpoint to keep it public and available to other areas in case of necessity.